packages/markdown/remark/src/types.ts

```diff
@@ -36,7 +37,8 @@ export type RemarkRehype = RemarkRehypeOptions;
 export type ThemePresets = BuiltinTheme | 'css-variables';
 
 export interface ShikiConfig {
-	langs?: LanguageRegistration[];
+	langs?: (LanguageRegistration | string)[];
```

Comment:
Extending this to a string is kinda a new feature I think, but I don't think it hurts extending it in this PR. Could we narrow down the types as
```suggestion
	langs?: (LanguageRegistration | BuiltinLanguage | SpecialLanguage)[];
```

So they get autocompletion for the types